asmcomp/amd64/arch.ml

```diff
@@ -42,6 +42,7 @@ type specific_operation =
   | Ibswap of int                      (* endianness conversion *)
   | Isqrtf                             (* Float square root *)
   | Ifloatsqrtf of addressing_mode     (* Float square root from memory *)
+  | Isextend                           (* Convert value with sign extension *)
 and float_operation =
```

Comment:
The name `Isextend` and the comment are not precise enough.  There are instructions to sign-extend 8, 16 and 32-bit quantities.

Reply:
Is `Isextend32` better?